services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You add more details to return type annotations in other places, so why don't you do the same here, for the sake of consistency? :)

Reply:
Aha, went a bit lazy here :)